src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -293,9 +293,9 @@ namespace cryptonote
     catch (...) { }
 
     BlockchainDB* db = nullptr;
-    uint64_t BDB_FAST_MODE = 0;
-    uint64_t BDB_FASTEST_MODE = 0;
-    uint64_t BDB_SAFE_MODE = 0;
+    uint64_t DBS_SAFE_MODE = MDB_NORDAHEAD;
+    uint64_t DBS_FAST_MODE = MDB_NORDAHEAD | MDB_NOSYNC;
+    uint64_t DBS_FASTEST_MODE = MDB_NORDAHEAD | MDB_NOSYNC | MDB_WRITEMAP | MDB_MAPASYNC;
```

Comment:
Seems a bit odd to set the MDB flags outside the if (db_type == "lmdb") case. Otherwise ok.